### PR TITLE
README update

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

- Bring up-to-date
- Update overview
- Add more configuration details
- Update maintainers
- Quick libp2p docs logo

## Latest preview

Check readme version on `readme-update`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,139 @@
-# libp2p Docs
+<p align="center">
+<img align="center" src="static/images/libp2p-docs-logo.png" width="615">
+</p>
+
+<div align="center">
+<h2> The networking layer of the web3 and the distributed web</h2>
```
Comment (marten-seemann) [inline README.md:6]: Can we remove this? libp2p is used much broader than just in web3.

File: README.md [hunk 2]
```diff
@@ -1,83 +1,141 @@
...
+It reflects the latest commit on `master` and publishes at [https://docs.libp2p.io](https://docs.libp2p.io).
+
+### Translation
+
+Please stay tuned for the steps to translate the documentation.
+
+## Primary maintainers
+
+- [@DannyS03](https://github.com/DannyS03): primary contact, project organization & technical writing(engineering)
+- [@mxinden](https://github.com/mxinden): libp2p steward(engineering), primarily rust-libp2p
```
Comment (marten-seemann) [inline README.md:132]: missing space?

File: README.md [hunk 3]
```diff
@@ -1,83 +1,141 @@
...
+- [@marten-seemann](https://github.com/marten-seemann): libp2p steward(engineering), primarily go-libp2p
+- [@MarcoPolo](https://github.com/MarcoPolo): libp2p steward(engineering), primarily go-libp2p
+- [@jennijuju](https://github.com/jennijuju): program management(engineering)
```
Comment (marten-seemann) [inline README.md:135]: I don't think this is accurate.

File: README.md [hunk 4]
```diff
@@ -1,83 +1,141 @@
...
+- [@p-shahi](https://github.com/p-shahi): libp2p project management(engineering)
+- [@BigLep](https://github.com/BigLep): interplanetary management and supervision(engineering)
+
+## License
 
-See [LICENSE file](./LICENSE) for details.
+All software code is copyright (c) Protocol Labs, Inc. under the **MIT license**. Other written documentation and content are copyright (c) Protocol Labs, Inc. under the [**Creative Commons Attribution-Share-Alike License**](https://creativecommons.org/licenses/by/4.0/).
```
Comment (marten-seemann) [inline README.md:141]: We actually use a dual license.

File: README.md (hunk 2 above)
Comment (DannyS03) [reply README.md]: addressed

File: README.md (hunk 1 above)
Comment (DannyS03) [reply README.md]: addressed

File: README.md (hunk 3 above)
Comment (DannyS03) [reply README.md]: Which part do you feel is inaccurate?

File: README.md (hunk 4 above)
Comment (DannyS03) [reply README.md:140]: Doesn't this mention the dual license?

Comment (marten-seemann) [reply README.md:140]: No. **The code** is licensed under two licenses, see https://protocol.ai/blog/announcing-the-permissive-license-stack/.

Comment (DannyS03) [reply README.md]: ah yes, updated

File: README.md (hunk 3 above)
Comment (DannyS03) [reply README.md]: updated

**Review by marten-seemann — APPROVED** [review]

**Comment by mxinden** [comment]
Thanks for the work on this @DannyS03 and @marten-seemann!

**Pull request merged**